release_notes/371-release-notes.adoc

```diff
@@ -0,0 +1,291 @@
+
+
+[id="network-policies-list"]
+=== List of network policies in Deployment tab for violations 
+
+A new information section has been added to help resolve a "missing Kubernetes network policy" violation. When viewing a violation, select the *Deployment* tab and scroll to the new *Network policy* section. It lists all the Kubernetes network policies applicable to the namespace of the offending deployment. You can use this list to verify which network policies exist in the namespace to help you determine why a deployment was flagged as missing a network policy. 
+
+
+[id="alpine-316"]
+=== Alpine 3.16 support
+Scanner now supports Alpine 3.16.
+
+== Enhancements
+
+[id="roxctl-image-scan"]
+=== Change to roxctl image scan behavior 
+
+The default value for the `--include-snoozed` option of the `roxctl image scan` command is set to `false`. If the `--include-snoozed` option is set to `false`, the scan does not include snoozed CVEs.
+
+[id="diagnostic-bundle-update"]
+=== Update to diagnostic bundles
+
+You can create diagnostic bundles to assist Red Hat in supporting you with {product-title-short}. This diagnostic bundle has been enhanced to include notifiers, auth providers and auth provider groups, access control roles with attached permission set and access scope, and system configuration information. Users with the `DebugLogs` permission can read listed entities from a generated diagnostic bundle regardless of their respective permissions. 
+
+
+[id="notable-tech-changes-371"]
+== Notable technical changes
+
+[id="ebpf-collection-default"]
+=== eBPF is now the default collection method
+
+In {product-title-short} 3.71, Red Hat has updated the default collection method for Collector to eBPF. Using eBPF for data collection takes advantage of modern kernel features and improves performance over kernel modules by 25%. Additionally, it reduces processed `syscall` events by 99%.
+
+
+[NOTE]
+====
+You should switch to kernel module collection if you are using:
+
+* {ocp} version 3.11, which uses {op-system-base-full} 7. It does not support the features required for performance improvement.
+* SELinux on {op-system-base} 7, which may also prevent the loading of eBPF probes.
+====
+
+[id="deprecation-notices-371"]
+== Deprecated and removed features
+
+Some features available in previous releases have been deprecated or removed.
+
+Deprecated functionality is still included in {product-title-short} and continues to be supported; however, it will be removed in a future release of this product and is not recommended for new deployments. For the most recent list of major functionality deprecated and removed, refer to the table below. Additional information about some removed or deprecated functionality is available after the table.
```

Comment:
@kcarmichael08 Should this be semicolon, or separate the independent clauses with a "." and promote them to sentences?

Reply:
It is copied & pasted from the OCP docs so I think it's OK